app/javascript/http_api/fetch.js

```diff
@@ -34,7 +34,7 @@ function processOptions(options) {
   }
 
   if (o.csrf) {
-    o.headers['X-CSRF-Token'] = $('meta[name=csrf-token]').attr('content');
+    o.headers['X-CSRF-Token'] = document.querySelector("meta[name=csrf-token]").getAttribute('content');
```

Comment:
`    TypeError: Cannot read property 'getAttribute' of null`

This has to be able to deal with the element not being present.